src/main/assisted-installer-controller/assisted_installer_main.go

```diff
@@ -38,5 +39,15 @@ func main() {
 		inventory_client.CreateInventoryClient(Options.ControllerConfig.ClusterID, Options.ControllerConfig.Host, Options.ControllerConfig.Port),
 		kc,
 	)
+
+	quit := make(chan bool)
+	go func() {
+		assistedController.ApproveCsrs(quit)
+	}()
 	assistedController.WaitAndUpdateNodesStatus()
+	logger.Infof("Sleeping for 10 minutes to give a chance to approve all crs")
+	time.Sleep(10 * time.Minute)
+
+	quit <- true
+	time.Sleep(assistedinstallercontroller.GeneralWaitTimeout)
```

Comment:
Wait for the go routine to complete

Reply:
quit <- true will wait till go routine will exit